### ISSUE TITLE
3-pane navigation is broken

### ISSUE THREAD

**Issue by obfischer** [opto/nostalgy-xpi#172]
According to the usage directives and to my current experience with Nostalgy++, the following shortcuts should work.

* ESC-m to focus the message pane
* ESC-f to focus the folder pan
* l to show the folder pan
* ESC-ESC to focus the message list

There are other shortcuts, I didn't test.

Currently these shortcuts does not work. 

So my first question is, if the 3-pane navigation is still supported?

If so, I would like to ask to to investigate my issue.

I am currenltly using TB 91.7.0 on MacOS and Nostagly++ 3.2.17

**Comment by opto** [comment]
yes, this works on win 11. (I just tested it a few days ago - all but l were tested).

I just tested on Linux and found that a log to console can take a really long time. If the same should happen on macOS, the timing could be broken if I left in some log statements.

I will prepare a version with all logs removed and will put it here for testing.

**Comment by obfischer** [comment]
Hi @opto, I will be happy to test it.

**Comment by opto** [comment]
thanks for offering. I am working with someone from the TB team to see what is happening ... there are just so many changes to Tb on the way to making TB more web/HTML app.
I followed some advice to put more of the new webextension code into Nostalgy++, but think that is too early, as the new and the old don't work together fine.
E.g., when using the new code for moving emails, one looses the possibility to undo - that just isn't implemented yet.

I will put up a test version soon,

Klaus

**Comment by opto** [comment]
you can test this: https://github.com/opto/nostalgy-xpi/releases/tag/v3.3.2

while logging a window is instantanious in win,i t took 10 sec in Ubuntu, holding up other code.
This has logs removed. Maybe they mixed up stuff?

Also, there is a new search with display for message and folder tree to DnD messages from search.
Please try, if you like, it is the 'browseraction button' with searchglas and ++.

Also, I noticed on Ubuntu that the new code puts the Nostalgy options in tools menu in win and in edit menu in Ubuntu, don't know where they will show on macOS.

**Comment by opto** [comment]
specifically, the esc shortcuts work on win and Ubuntu, I forgot to test the l. Also S, shift S and G were tested

**Comment by opto** [comment]
TB team (John) helped to optimise startup, I am interested whether this solves the problems.
https://github.com/opto/nostalgy-xpi/releases/tag/v3.3.17

Klaus

**Comment by obfischer** [comment]
Sorry, I was to busy to remember. I will test it today.

**Comment by obfischer** [comment]
Hi @opto, no, I don't see any difference to version 3.3.16.

**Comment by opto** [comment]
I am just thinking: the code expects the two keys within a certain time. If the timer calibration is off for a Mac, that might be a reason why you cannot do esc-m or similiar.

1) you could try to do it very fast, just as a test
2)if that does not help, I can make that delay to be set in options.

**Comment by opto** [comment]
there is a new version in releases but I don't think it will help this specific problem

**Comment by obfischer** [comment]
>2) if that does not help, I can make that delay to be set in options.

I tried to hit the keys as fast as possible, but without any success. Making the delay configurable could be a way to solve it?

Does the feature works on you machine, which is a Windows system, right?

**Comment by opto** [comment]
works both on windows and on Ubuntu, that is the strange thing. I think I will enter a configurable delay in options

**Comment by obfischer** [comment]
Let me check this problem on a new user account on my computer. Maybe the problem is caused not by the addon, but by another piece of software. I will report the result here.

**Comment by obfischer** [comment]
Lets us wait, before you start. I will setup a new account on my machine 
and test this issue there. Maybe the reason for this problem is caused 
by another peace of software, running on my computer. Simply to avoid 
unneeded work.

**Comment by obfischer** [comment]
Hi @opto,

Sorry for the long delay. Today I was able to check this on another MacBook there I only have Nostalgy++ installed. This installation was working sometimes, but sometimes I didn't saw any effect. Sometimes I also activated the search input box of Thunderbird. 

I didn't recognize a clear pattern and therefore it is difficult to describe. The only thing I can offer is to record a small screencast and to describe you, what I do. 

What do you thing?
